browser/js/kit/kit.controllers.js

```diff
@@ -1,8 +1,30 @@
-app.controller('KitsController', function ($scope, allKits, allKitImages) {
-	
+app.controller('KitsController', function ($scope, allKits, allKitImages, AuthService, $uibModal, $log) {
+	AuthService.getLoggedInUser().then(function(user){
+		$scope.user= user;
+	})
 	$scope.kits = allKits
 	$scope.images = allKitImages
 	$scope.title= 'All Kits'
+	$scope.open = function (size) {
+	    var modalInstance = $uibModal.open({
+	      animation: $scope.animationsEnabled,
+	      templateUrl: './js/modalWindow/modal.html',
+	      controller: 'ModalInstanceCtrl',
+	      size: size,
+	      // resolve: {
```

Comment:
@elizabethkim828 just pushed it up so you guys can see what was going on, I'm gonna run through everything and remove all comments and logs.